play_scraper/utils.py

```diff
@@ -250,8 +251,8 @@ def parse_app_details(soup):
     # Let the user handle modifying the URL to fetch different resolutions
     # Removing the end `=w720-h310-rw` doesn't seem to give original res?
     # Check 'src' and 'data-src' since it can be one or the other
-    screenshots = [parse_screenshot_src(img)
-                   for img in soup.select('button.NIc6yf img.lxGQyd')]
+    screenshots = [parse_screenshot_src(btn)
```

Comment:
Seems like this is broken - either change both to `btn` or simply leave it as `img`.